### ISSUE TITLE
Does not work with nbconvert 6.x

### ISSUE THREAD

**Issue by lanking520** [greenape/mknotebooks#46]
Recently nbconvert released 6.0.2 and it removed `celltags.tpl`.  The current 0.4.1 does not work with that.

```
  File "/opt/hostedtoolcache/Python/3.8.5/x64/lib/python3.8/site-packages/jinja2/_compat.py", line 28, in reraise
    raise value.with_traceback(tb)
  File "/opt/hostedtoolcache/Python/3.8.5/x64/lib/python3.8/site-packages/mknotebooks/templates/custom_markdown.tpl", line 2, in top-level template code
    {% from 'celltags.tpl' import celltags %}
jinja2.exceptions.TemplateNotFound: celltags.tpl
```

**Comment by greenape** [comment]
Hmm. Looks there were quite a few changes for the 6.0.0 release that will require changes here too. What I'll do, is cut a new release today and add an upper limit on nbconvert version, then fix up for another release after that.

**Comment by greenape** [comment]
This should be fixed by 0.6.0.

**Comment by lanking520** [comment]
```  
File "/opt/hostedtoolcache/Python/3.8.5/x64/lib/python3.8/site-packages/markdown/extensions/md_in_html.py", line 78, in run
    block = self._process_nests(element, block)
  File "/opt/hostedtoolcache/Python/3.8.5/x64/lib/python3.8/site-packages/markdown/extensions/md_in_html.py", line 45, in _process_nests
    self.run(element, block[nest_index[-1][0]:nest_index[-1][1]],  # last
  File "/opt/hostedtoolcache/Python/3.8.5/x64/lib/python3.8/site-packages/markdown/extensions/md_in_html.py", line 52, in run
    tag = self._tag_data[self.parser.blockprocessors.tag_counter]
IndexError: list index out of range
```

Still have some issues after pulling the changes
